projects/demo/src/app/demo/pages/toggle-page/toggle-page.component.html

```diff
@@ -0,0 +1,53 @@
+<pa-demo-page>
+    <pa-demo-title>Toggle</pa-demo-title>
+    <pa-demo-description>Pa field (with label and help) for a checkbox with a "slide toggle" design.</pa-demo-description>
+
+    <pa-demo-examples>
+        <h4>Using ngModel ({{firstToggle}})</h4>
+        <pa-toggle
+            id="firstToggle"
+            name="firstToggle"
+            [(ngModel)]="firstToggle"
+            [hasFocus]="focusedFirstToggle"
+            [disabled]="disabled"
+        ></pa-toggle>
+
+        <h4>Using formControl ({{form.value.toggle}})</h4>
+        <form [formGroup]="form">
+            <pa-toggle
+                id="toggle"
+                name="toggle"
+                formControlName="toggle"
+                [help]="'help text'"
```

Comment:
no need for `[help]` you can do `help="help text"`

Reply:
sure